src/lib/viewers/BaseViewer.js

```diff
@@ -690,7 +690,8 @@ class BaseViewer extends EventEmitter {
             fileVersionId,
             isMobile: this.isMobile,
             locale: location.locale,
-            previewUI: this.previewUI
+            previewUI: this.previewUI,
+            modeButtons: ANNOTATION_BUTTONS
```

Comment:
This will allow us to pass in any selector which will indicate the annotation button css selector in Preview aka ultimately platform and webapp can use different annotation button icons.